lib/puppet/provider/rabbitmq_common.rb

```diff
@@ -0,0 +1,44 @@
+class Puppet::Provider::Rabbitmq_common < Puppet::Provider
```

Comment:
Ypu are right, the rabbitmq_common should be reimplemented as a part of Puppet::Provider::Rabbitmqctl.